From f28ca8db34fa38f5cdb9142196682c7211b93189 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Sat, 8 Aug 2020 18:57:47 +0200
Subject: [PATCH 1/1] lib/crypto: simplify public_key_verify_signature

The variable region is filled but never used. Remove it.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 lib/crypto/public_key.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/lib/crypto/public_key.c b/lib/crypto/public_key.c
index a8f7fbed45..df6033cdb4 100644
--- a/lib/crypto/public_key.c
+++ b/lib/crypto/public_key.c
@@ -97,7 +97,6 @@ int public_key_verify_signature(const struct public_key *pkey,
 				const struct public_key_signature *sig)
 {
 	struct image_sign_info info;
-	struct image_region region;
 	int ret;
 
 	pr_devel("==>%s()\n", __func__);
@@ -137,9 +136,6 @@ int public_key_verify_signature(const struct public_key *pkey,
 	info.key = pkey->key;
 	info.keylen = pkey->keylen;
 
-	region.data = sig->digest;
-	region.size = sig->digest_size;
-
 	if (rsa_verify_with_pkey(&info, sig->digest, sig->s, sig->s_size))
 		ret = -EKEYREJECTED;
 	else
-- 
2.28.0

